src/js/game/LevelMVC/LevelView.js

```diff
@@ -573,7 +576,21 @@ module.exports = class LevelView {
   playOpenChestAnimation(position) {
     const blockIndex = (this.yToIndex(position[1])) + position[0];
     const block = this.actionPlaneBlocks[blockIndex];
-    this.playScaledSpeed(block.animations, "open");
+    const animation = this.playScaledSpeed(block.animations, "open");
+    this.onAnimationEnd(animation, () => {
+      const treasure = this.getTreasureTypeFromChest(this.controller.levelModel.actionPlane.getBlockAt(position));
+      if (treasure) {
+        this.createMiniBlock(position[0], position[1], treasure, {
+          collectibleDistance: 1,
+          xOffsetRange: 10,
+          yOffsetRange: 10
```

Comment:
Let's override these both to 0, so the item always spawns centered on the open chest.